x-pack/platform/plugins/shared/features/server/config.test.ts

```diff
@@ -39,7 +39,7 @@ describe('config schema', () => {
       ConfigSchema.validate(
         {
           overrides: {
-            featureA: { name: 'new name', hidden: true },
+            featureA: { name: 'new name', description: 'new description', hidden: true },
```

Comment:
If this logic does stay in this plugin, tests should be added for the new `null` case

Reply:
done